desktop/app.js

```diff
@@ -1,28 +1,32 @@
 'use strict';
 
+let authState,
+  authWindow,
+  mainWindow = null;
```

Comment:
not important, just a note of style, we have put `let`s one per line in Calypso. `var`s are still combined to match the semantic of hosting, but not `let`